app/assets/v2/js/pages/carousel.js

```diff
@@ -1,28 +1,29 @@
-const slides = document.querySelectorAll('#slides .slide');
+const slides = document.querySelectorAll( '#slides .slide' );
+
 let currentSlide = 0;
-if(slides.length){
-  slides[0].className = 'slide showing';
 
-  function nextSlide() {
-    slides[currentSlide].className = 'slide';
-    currentSlide = (currentSlide+1) % slides.length;
-    slides[currentSlide].className = 'slide showing';
-    resetTimer();
-  }
+function nextSlide() {
+  slides[currentSlide].className = 'slide';
+  currentSlide = ( currentSlide+1 ) % slides.length;
+  slides[currentSlide].className = 'slide showing';
+  resetTimer();
+}
 
-  function prevSlide() {
-    slides[currentSlide].className = 'slide';
-    if(currentSlide == 0)
-      currentSlide = slides.length - 1;
-    else
-      currentSlide = (currentSlide - 1) % slides.length;
-    slides[currentSlide].className = 'slide showing';
-    resetTimer();
-  }
+function prevSlide() {
+  slides[currentSlide].className = 'slide';
+  if ( currentSlide == 0 )
+    currentSlide = slides.length - 1;
+  else
+    currentSlide = ( currentSlide - 1 ) % slides.length;
+  slides[currentSlide].className = 'slide showing';
+  resetTimer();
+}
 
-  function resetTimer() {
-     clearInterval(interval);
-     startTimer();
-  }
-  
+function resetTimer() {
```

Comment:
Thoughts about rewriting it as `const resetTimer = () => { }` ? Yay or Nay ?

Reply:
Yea we could since it is contained in that file only.  When we do imports maybe will switch back to function so can be exported if it is used elsewhere.

I'm not sure if the plan is to reuse the function....thoughts?

Reply:
Ah we could still use it via an import right ? Maybe I'm not getting your question right
`resetTimer` can be exported either way it's defined right?

Reply:
Yes it can be imported either way.  As it stands now we should be able to import by doing

`export { resetTimer }` or `export default resetTimer }`

I think functions should stay functions unless there is a reason to change them to **consts**?  There very well could be, not expert with es6 yet :)

Reply:
Ah in that case we'll leave it as it is ^_^